packages/cloudflare/src/cli/build/patches/to-investigate/inline-next-require.ts

```diff
@@ -1,30 +1,30 @@
 import { existsSync, readFileSync } from "node:fs";
 import { join } from "node:path";
 
-import { Config } from "../../../config.js";
+import { type BuildOptions, getPackagePath } from "@opennextjs/aws/build/helper.js";
 
 /**
  * The following avoid various Next.js specific files `require`d at runtime since we can just read
  * and inline their content during build time
  */
-export function inlineNextRequire(code: string, config: Config) {
-  const pagesManifestFile = join(config.paths.output.standaloneAppServer, "pages-manifest.json");
-  const appPathsManifestFile = join(config.paths.output.standaloneAppServer, "app-paths-manifest.json");
+// TODO(vicb): __NEXT_PRIVATE_RUNTIME_TYPE is not handled by this patch
```

Comment:
could you create an issue or provide more context to this todo comment?

Reply:
I didn't before I have a PR ready, only need to be rebased and submitted.